js/script.js

```diff
@@ -35,18 +37,38 @@ $(locations).click(
 	function(){
 		$('#locationPopup').css("visibility", "visible");
 		var thisAudio = ("audio/"+this.id+".mp3");
-		var thisPhoto1 = ("images/"+this.id+"-1.jpg");
-		var thisPhoto2 = ("images/"+this.id+"-2.jpg");
-		var thisPhoto3 = ("images/"+this.id+"-3.jpg");
 		var speaker = $(this).data("speaker");
 		var count = $(this).data("count");
+
+		// empty out pager and slideshow
+		$('.dot').remove();
+		$('.mySlides').remove();
+
+		// helper function to deal with for-loop scope of `i`
```

Comment:
for more info: https://stackoverflow.com/questions/4091765/assign-click-handlers-in-for-loop